### PR TITLE
Modified guide according to new feedback

### PR DESCRIPTION
**Pull request by Beak-man**
Various enhancements to the Get Started guide, according to @emxsys' feedback:
 - Homogeneously changed how we refer to files and folders. All of them are referred to as 'file.js' and '\folder' (enclosed within single quotes and a forward slash in the case of folders).
- Added links to the related API doc to the various objects described in the guide ("WorldWindow", "Layer", "Placemark", "Polygon", etc).
- Avoided displaying altitudes in exponential notation. It saves us from potential confusions and explanations on it.
- Every latitude, longitude and altitude has a '.0' appended at the end, just to convey that they're not integer values.
- Explained usage of geographic coordinates a bit better, and provided a link for it.
- Explained what COLLADA models are.
- Invited reader to play around with the tutorial "Try putting a placemark over your house". Maybe more invites like this are a good idea?
- Various enhancements to wording and text contents.

Potential disadvantages:
- This change implies the need of manually maintaining and updating the links to our API docs.

### CODE REVIEW

File: content/web/get-started.md [hunk 1]
```diff
@@ -7,12 +7,23 @@ draft: false
...
+- • Loading and displaying a WorldWind globe
+- • Setting up a development web server
+- • Displaying some objects over the globe (placemarks and 3D shapes)
+- • Displaying imagery from a 3rd party map service
+
+---
+
+### Loading and displaying a WorldWind globe
+
+There are multiple ways to load the Web WorldWind library. The simplest is to include a script tag in an HTML page to link it directly from WorldWind's servers. Let's create a file named 'index.html' and add the following to it:
```
Comment (emxsys) [inline content/web/get-started.md]: Let's remove any ambiguity about where to create files.  Perhaps introduce the concept of a "project folder" that you can refer to later if needed. For instance:  "Let's create a file named 'index.html' in the project folder and add the following to it: ..."

File: content/web/get-started.md [hunk 2]
```diff
@@ -60,28 +71,26 @@ wwd.addLayer(new WorldWind.ViewControlsLayer(wwd));
...
 
 If you do not already have a web server, you can use a minimal development web server built in [Node.js](https://en.wikipedia.org/wiki/Node.js):
 
-- - Download and install [Node](https://www.nodejs.org).
-- - Open a command line console in the root folder of your web app (where the **index.html** file and your _images_ folder is located).
-- - In the command line console, run `npm install http-server`.
-- - Execute the command `http-server` to start the server.
-- - In your web browsear, head to the address http://localhost/:8080.
+- • Download and install [Node](https://www.nodejs.org).
+- • Open a command line console in the root folder of your web app (where the 'index.html' file and your '/images' folder is located).
```
Comment (emxsys) [inline content/web/get-started.md]: I don't agree with the '/folder' convention.  It's literally path off the root.  But I understand your intent. You could proceed the folder with './' (e.g., './folder' ) which is a path within the current (project?) folder.

File: content/web/get-started.md [hunk 3]
```diff
@@ -157,16 +166,16 @@ placemarkAttributes.imageSource = WorldWind.configuration.baseUrl + "images/push
...
 
-We will use a `WorldWind.Position` object to define the placemark location in terms of latitude, longitude, and altitude (note that the altitude in this case is defined in exponential notation, _e.g._ 1e2 = 100 meters).
+We will use a [WorldWind.Position](https://nasaworldwind.github.io/WebWorldWind/Position.html) object to define the placemark location over the globe in terms of [geographic coordinates](https://en.wikipedia.org/wiki/Geographic_coordinate_system): latitude, longitude, and altitude. Most placements in WorldWind are defined in terms of geographic coordinates.
 
 Secondly, we'll define if we desire the placemark size to scale according to the camera zoom. In this case we want the placemark size to remain constant, so it's set to `false`.
 
 The last argument binds the previously created `PlacemarkAttributes` to our placemark.
 
-Our placemark creation looks like:
+Hence, the next steps on our placemark creation look like:
```
Comment (emxsys) [inline content/web/get-started.md]: Please consider this edit "next steps to create a placemark at 55 degrees latitude and 106 degrees longitude at 100 meters ....."  (use degree symbols if you can).

File: content/web/get-started.md [hunk 4]
```diff
@@ -179,6 +188,8 @@ placemark.label = "Placemark\n" +
 placemark.alwaysOnTop = true;
 {{< /highlight >}}
 
+Experiment with the placemark positioning and customization. Try putting a placemark at your house, for instance.
```
Comment (emxsys) [inline content/web/get-started.md]: 👍

**Comment by Beak-man** [comment]
@emxsys Thanks a lot for your input, I think every suggestion is spot on, and I followed them all. Please review the changes at your convenience.

File: content/web/get-started.md (hunk 2 above)
Comment (Beak-man) [reply content/web/get-started.md]: Good observation. The convention is changed.

File: content/web/get-started.md (hunk 3 above)
Comment (Beak-man) [reply content/web/get-started.md]: Done, and the ° symbol seems to work OK in Hugo/Markdown.

**Comment by Beak-man** [comment]
Right! I totally forgot that as well.

**Comment by Beak-man** [comment]
@emxsys it's done. I changed the wording of "altitude" for "elevation" as well.

**Pull request merged**